packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -153,7 +153,7 @@ contract GoldToken is
   function transferFrom(address from, address to, uint256 value) external returns (bool) {
     require(to != address(0), "transfer attempted to reserved address 0x0");
     require(
```

Comment:
CeloDistributionSchedule distributes rewards based on its own balance. If the balance is changed from outside by sending Celo to this contract, the distribution schedule will redistribute also this amount. Since we are removing CeloDistributionSchedule logic we will also remove this part.